LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 PurpleMyst
+Copyright (c) 2024 PurpleMyst
```

Comment:
@PurpleMyst do you want this to be something like `2018-2024`?

Reply:
Sure, that'd be good for me.